mixer/pkg/attribute/protoBag.go

```diff
@@ -64,7 +64,6 @@ var protoBags = sync.Pool{
 // When you are done using the proto bag, call the Done method to recycle it.
 func GetProtoBag(proto *mixerpb.CompressedAttributes, globalDict map[string]int32, globalWordList []string) *ProtoBag {
 	pb := protoBags.Get().(*ProtoBag)
-
```

Comment:
i'd just revert this change. no sense in touching this file just for cleanup of an empty line.